client/gutenberg/extensions/tiled-gallery/layout/index.js

```diff
@@ -97,3 +105,48 @@ export default class Layout extends Component {
 function isSquareishLayout( layout ) {
 	return [ 'circle', 'square' ].includes( layout );
 }
+
+function isWpcomFilesUrl( url ) {
+	const { host } = parseUrl( url );
+	return /\.files\.wordpress\.com$/.test( host );
+}
+
+/**
+ * Apply photon arguments to *.files.wordpress.com images
+ *
+ * This function largely duplicates the functionlity of the photon.js lib.
+ * This is necessary because we want to serve images from *.files.wordpress.com so that private
+ * WordPress.com sites can use this block which depends on a Photon-like image service.
+ *
+ * If we pass all images through Photon servers, some images are unreachable. *.files.wordpress.com
+ * is already photon-like so we can pass it the same parameters for image resizing.
+ *
+ * @param  {string} url  Image url
+ * @param  {Object} opts Options to pass to photon
+ *
+ * @return {string}      Url string with options applied
+ */
+function photonWpcomImage( url, opts ) {
```

Comment:
I think you accidentally removed the default value for `opts` again in 3b436407ad980c201103a600b1b415099c0cdd89

Reply:
Thanks again, fixed this.